packages/expo-calendar/src/Calendar.ts

```diff
@@ -974,6 +979,14 @@ export async function updateEventAsync(
     throw new Error('updateEventAsync must be called with an id (string) of the target event');
   }
 
+  if (Platform.OS === 'android') {
+    if (details.hasOwnProperty('color')) {
+      console.warn(
+        'updateEventAsync was called with a read-only property, which will not be updated'
```

Comment:
Lets make sure people know which property we are warning about 😄 
```suggestion
        'updateEventAsync was called with a read-only property `color`, which will not be updated'
```